base/intfuncs.jl

```diff
@@ -198,7 +198,7 @@ end
 function ndigits0z(n::Unsigned, b::Int)
     d = 0
     if b < 0
-        d = ndigits0znb(signed(n), b)
+        d = ndigits0znb(n, b)
     else
```

Comment:
I had tried that too, but I guess it didn't do it systematically enough. Adding a timeout on individual tests would indeed make this much clearer...